charts/steward/templates/deployment-tenant-controller.yaml

```diff
@@ -34,6 +34,8 @@ spec:
         imagePullPolicy: {{ .pullPolicy | quote }}
         {{- end }}
         args:
+        - {{ printf "-qps=%d" ( .Values.runController.args.qps | int ) | quote }}
```

Comment:
Copy paste error? The values from the runController config are used!
@LenaHoinkis